src/mg/PAMI/Client/Impl/ClientImpl.php

```diff
@@ -282,7 +282,11 @@ public function process()
                 $bMsg .= 'ActionId: ' . $this->_lastActionId . "\r\n" . $aMsg;
                 $event = $this->_messageToEvent($bMsg);
                 $response = $this->findResponse($event);
-                $response->addEvent($event);
+                if ($response === false) {
```

Comment:
Hi - Honestly I would need some time to collect more details, but I've just noticed that an application I've built that uses your library throws this error by time to time
```
E_ERROR: Call to a member function addEvent() on a non-object
```

this is a portion of the stack trace
```
…ses/20150331120320/vendor/marcelog/pami/src/mg/PAMI/Client/Impl/ClientImpl.php (285)
…ses/20150331120320/vendor/marcelog/pami/src/mg/PAMI/Client/Impl/ClientImpl.php (410)
…150331120320/src/Facile/Ws/BunnyBundle/Controller/Utils/AsteriskController.php (110)
```